bookdown/preamble.tex

```diff
@@ -88,3 +86,5 @@
 \urlstyle{tt}
 
 \makeatother
+
+ \renewenvironment{Shaded} {\begin{snugshade}\small} {\end{snugshade}}
```

Comment:
What is this about?

Reply:
Code block resizing (`\small`). 

`snugshade` is the env for the grey background of code blocks.